# HG changeset patch
# User Jacob Dawid <jacob.dawid@gmail.com>
# Date 1342639746 14400
#      Wed Jul 18 15:29:06 2012 -0400
# Node ID 76452c97add8ccdb0794dd3c0ca82dfdd429b33d
# Parent  a2e3bb987929a400e1fb3d86ab4fa574be4ad3cd
Removed restriction on maximum terminal size.

diff --git a/libqterminal/unix/TerminalView.cpp b/libqterminal/unix/TerminalView.cpp
--- a/libqterminal/unix/TerminalView.cpp
+++ b/libqterminal/unix/TerminalView.cpp
@@ -183,17 +183,17 @@ void TerminalView::fontChange(const QFon
 
 
   if (_fontWidth < 1)
     _fontWidth=1;
 
   _fontAscent = fm.ascent();
 
   emit changedFontMetricSignal( _fontHeight, _fontWidth );
-  parentWidget()->setFixedWidth(_fontWidth * 80 + _leftMargin);
+  //parentWidget()->setFixedWidth(_fontWidth * 80 + _leftMargin);
   propagateSize();
   update();
 }
 
 void TerminalView::setVTFont(const QFont& f)
 {
   QFont font = f;
 
